snap/local/stub-chown/stub-chown.c

```diff
@@ -0,0 +1,10 @@
+#include <sys/types.h>
+#include <stdio.h>
+
+int chown(const char *pathname,
+          uid_t owner,
+          gid_t group)
+{
+  fprintf(stderr, "stub-chown: Stubbed out attempt to chown '%s'\n", pathname);
+  return 0;
+}
```

Comment:
I don't really remember what problem this solved. Something crashed/didn't want to work because it tried to chown things it wasn't allowed to access due to apparmor rules. I'd just leave it there.